plugins/crd/handler/customnetwork/custom_network_handler.go

```diff
@@ -98,10 +99,51 @@ func (h *Handler) customNetworkToProto(customNetwork *v1.CustomNetwork) *model.C
 		customNetworkProto.Type = model.CustomNetwork_L2
 	case "L3":
 		customNetworkProto.Type = model.CustomNetwork_L3
-	case "stub":
+	case "STUB":
```

Comment:
Maybe it is github but I do not see the stub->STUB update reflected in the examples.
I guess it works anyway though, since CustomNetwork_STUB is used if the above switch has no match, correct?
Not sure if and where we could throw error when a pod's annotation is incorrect.

Reply:
Yeah, this was related just to the CRD. Actually, I think that I will `STUB` custom network option completely, as there is no need for having multiple stub networks.